### PR TITLE
Updating GitHub & Twitter profile links

### PR DESCRIPTION
**Pull request by s2sharpit**
## Related Issue

- Information about the related issue

Closes: #306 

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by siddhi-244 — APPROVED** [review]

**Review by darkterminal — APPROVED** [review]
Like I said before, this is nice catch!!! :tada:

File: config/site.ts [hunk 1]
```diff
@@ -27,10 +26,10 @@ export const siteConfig = {
     },
   ],
   links: {
-    twitter: "https://twitter.com/shadcn",
-    github: "https://github.com/shadcn/ui",
-    docs: "https://ui.shadcn.com",
+    twitter: "https://twitter.com/WebXDAO",
+    github: "https://github.com/WebXDAO/WebXDAO.github.io",
```
Comment (vinzvinci) [inline config/site.ts]: use this link https://github.com/WebXDAO

Comment (vinzvinci) [reply config/site.ts]: done xD

**Review by vinzvinci — APPROVED** [review]
LGTM! Merging this now!

**Pull request merged**